AutoMapper.Data.Tests/ProfileTests.cs

```diff
@@ -0,0 +1,147 @@
+﻿namespace AutoMapper.Data.Tests
+{
+    using System.Collections.Generic;
+    using System.Data;
+    using System.Linq;
+    using System.Reflection;
+    using AutoMapper.Data.Configuration.Conventions;
+    using Shouldly;
+    using Xunit;
+
+    public abstract class ProfileTestsBase
+    {
+        public ProfileTestsBase()
+        {
+            Mapper.Reset();
```

Comment:
You should use the instance based API here. The static thing will be gone in the next AM version.

Reply:
Good to know. I'll submit a PR for this shortly.